vignettes/introduction-esqlabsr.Rmd

```diff
@@ -47,3 +77,11 @@ resultOfAFunction <- object1$multiply(1,2)
 ```
 
 Important information about the object can be printed out by calling `print(object)`.
+
+## Function reference
```

Comment:
There is already an automatically generated function reference, I think this should be removed.